api/v1/models/Attendee.js

```diff
@@ -45,20 +45,19 @@ const Attendee = Model.extend({
     isNovice: ['required', 'boolean'],
     isPrivate: ['required', 'boolean'],
     phoneNumber: ['string', 'maxLength:15'],
-    acceptanceType: ['string', validators.in(ACCEPTANCE_TYPES)],
-    acceptedEcosystemId: [ 'integer' ]
+    acceptanceType: ['string', validators.in(ACCEPTANCE_TYPES)]
```

Comment:
We don't need this field, we don't have two separate create/contribute tracks anymore

Reply:
Removed field in 1520608d078fb1bbcfa15d2e7ea164d0675f3c80